crates/interpreter/src/side_table.rs

```diff
@@ -65,15 +61,15 @@ impl<'m> Metadata<'m> {
     }
 
     pub fn branch_table(&self) -> &[BranchTableEntry] {
-        bytemuck::cast_slice(&self.0[5 ..])
+        bytemuck::cast_slice(&self.0[10 ..])
     }
 
     pub fn parser_range(&self) -> Range<usize> {
-        self.read_u32(1) .. self.read_u32(3)
+        self.read_u32(2) .. self.read_u32(6)
     }
 
     fn read_u32(&self, idx: usize) -> usize {
-        bytemuck::pod_read_unaligned::<u32>(bytemuck::cast_slice(&self.0[idx .. idx + 2])) as usize
+        u32::from_le_bytes(self.0[idx .. idx + 4].try_into().unwrap()) as usize
```

Comment:
similarly we could move this to a parse_u32 (however here we can't take an index, we must take an offset, so probably we should also continue taking an offset for parse_u16)